### PR TITLE
[DEV-2047] remove buyFiat outputAsset string cols

### PR DESCRIPTION
**Pull request by Yannick1712**
#### Pre-Release

- [ ] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations (on [DFX services](https://dev.services.dfx.swiss))
  - Login/logout
  - Buy/sell payment request
  - KYC page

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

**Comment by Yannick1712** [comment]
Don't merge before merging #1130

**Review by davidleomay — APPROVED** [review]

**Pull request merged**
